### ISSUE TITLE
Code implementation is different from the paper

### ISSUE THREAD

**Issue by Xinpeng-Wang** [intersun/CoDIR#5]
Hi, Siqi
Nice work! 
In the part of selecting teacher features for comparison, the code shows that the teacher features are selected from every two layers, which is different from the paper ( concat all layers and project to lower dimension ). And the result I got with contrastive loss shows only little improvement, which is different from the paper. 

Could you give some hint about your training and the hyperparam setting?

**Comment by jinxinglu** [comment]
HI, Xinpeng, how did you do the implement of the negative examples in the pre-training? I can't find that from the paper and the code

thanks
